src/pages/index.js

```diff
@@ -75,22 +131,64 @@ addCardButton.addEventListener("click", () => {
   cardPopup.open();
 });
 
-// CLASSES
-const profileEditPopup = new PopupWithForm(profileEditModal, handleFormSubmit);
+profileImageEditButton.addEventListener("click", () => {
+  profileImageEditPopup.open();
+})
+
+
+
+// ~~~~~~~~~~~~~~~~~ API ~~~~~~~~~~~~~~~~~ //
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "ba6f95f2-7e3a-4192-b9e4-9397fc618de9",
+    "Content-Type": "application/json"
+  }
+});
+
+let cardSection;
+
+api.getInitialCards()
+.then((cards) => {cardSection = new Section({
+  items: cards,
+  renderer: createCard,
+}, cardListEl);
+cardSection.renderItems();
```

Comment:
Needs correction: It's important to format code correctly, use the indentation hierarchy, and split your code. Please check these sources to study the standard: [https://code.s3.yandex.net/web-developer/landings/design-rules-en/index.html](https://code.s3.yandex.net/web-developer/landings/design-rules-en/index.html), [https://javascript.info/coding-style](https://javascript.info/coding-style).

To make sure your code is formatted correctly, we suggest using Prettier (https://prettier.io/). It can be integrated with any code editor or IDE.

You should fix the formatting in all the files